docs/makeup-listbox-button/index.html

```diff
@@ -324,7 +324,60 @@ <h3>Unselected, Floating label</h3>
             </div>
           </div>
         </div>
-        
+
+        <h3>Unselected - with descripton</h3>
```

Comment:
Can we add an example to the Listbox page also, please.

Reply:
I was initially considering to add it, but there was no visual difference, which is why I did not add it.
I did add it now just for consistency.